lib/src/customer_io.dart

```diff
@@ -111,7 +111,9 @@ extension AmiAppSDKExtensions on CustomerIOSDK {
   Future<String?> getBuildInfo() async {
     try {
       PackageInfo packageInfo = await PackageInfo.fromPlatform();
-      return 'Customer.io Flutter SDK 1.2.1 ${packageInfo.appName} ${packageInfo.version} (${packageInfo.buildNumber})';
+      // TODO Fetch version from the SDK
+      String sdkVersion = '1.2.1';
+      return 'Customer.io Flutter SDK $sdkVersion ${packageInfo.appName} ${packageInfo.version} (${packageInfo.buildNumber})';
```

Comment:
Since we are trying to get sample apps all to be the same, are we OK with merging these changes without having this feature of getting the SDK version done? 

I don't want to merge it and we all think that sample app work is done and realize there is more work to do here.

Reply:
Agreed. I figured out we can use `customer_io_plugin_version` to fetch version from the package, [updated the code](https://github.com/customerio/amiapp-flutter/pull/12/commits/da126ff88bee1ed8cb15d407b97c8144d2f18146) to use it for exact version.